### ISSUE TITLE
How You Reported Vulnerability to IOBIT ?

### ISSUE THREAD

**Issue by RashidKhanPathan** [tomerpeled92/CVE#1]
Hello i Just Wanted to Know How You Have Reported Vulnerabilities to IOBIT, Because i Couldn't Find any Email or Bug Report Platform From IOBIT ? Or Did You Just First Requested CVE for These Vulnerabilities ?
Waiting for Your Response

**Comment by tomerpeled92** [comment]
I contacted their customer service first.
then i asked my emails will be transferred to their security personal.
they refused to do that or give me a direct line to their security personal, instead they said they will transfer all the information themselves and contact me back.
they never did... after 3 months we contacted them back and they said they'll look into that we sent our post, a few weeks after no further emails we decided to go ahead with publishing our findings
